name.abuchen.portfolio/src/name/abuchen/portfolio/model/ImageManager.java

```diff
@@ -22,12 +22,37 @@ private ImageManager()
     {
     }
 
+    /**
+     * Retrieves an image associated with the given Attributable and AttributeType.
+     * If not found, a default image is returned.
+     *
+     * @implNote For some time, Windows cut off the logos on the left.
+     *           An offset was added with #2752 In the meantime the problem seems to be fixed,
+     *           so we set the offset to 0 again.
+     *
+     * @see <a href="https://github.com/portfolio-performance/portfolio/pull/2752">#2752</a>
+     *
+     * @param target The Attributable object to retrieve the image for.
+     * @param attr   The AttributeType specifying the image.
+     * @return The retrieved image or null if not found.
+     */
     public Image getImage(Attributable target, AttributeType attr)
     {
-        int xOffset = Platform.OS_WIN32.equals(Platform.getOS()) ? 1 : 0;
+        int xOffset = Platform.OS_WIN32.equals(Platform.getOS()) ? 0 : 0;
```

Comment:
If the x offset is not needed anymore, then we can completely remove this statement as it is always zero.

Reply:
Done